### PR TITLE
Fix the tests, and set up GitHub Actions CI

### PR DESCRIPTION
**Pull request by DilumAluthge**

### CODE REVIEW

File: .github/workflows/CompatHelper.yml [hunk 1]
```diff
@@ -1,24 +0,0 @@
-name: CompatHelper
```
Comment (giordano) [inline .github/workflows/CompatHelper.yml:1]: Why?

Comment (DilumAluthge) [reply .github/workflows/CompatHelper.yml:1]: Whoops 😆 

Restored.

**Comment by giordano** [comment]
Cirrus can probably removed, last time I tried it was just unusable

**Comment by DilumAluthge** [comment]
> Cirrus can probably removed, last time I tried it was just unusable

Done.

**Pull request merged**